spec/services/encryptor_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
Because it covers the decryption path I think this test hides the 'missing decryption test' (see other comments).

Reply:
@starswan I'm afraid I disagree: this is testing encryption and decryption, as opposed to reading an encrypted field in the db and automatically decrypting it.  I.e. it seems perfectly legitimate to test encryption outside of the database.

 Could you clarify what the issue is? Perhaps a quick call would help?

Reply:
I'm not sure that it tests enough. If you disagree that's fine, I'm probably wrong

Reply:
sorry my mistake - combination of mis-reading several things.